FirebaseInAppMessaging/Sources/Analytics/Protogen/nanopb/fiam.nanopb.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 Google
```

Comment:
Do you know why this shows up as new and not deleted?  Does the original still need to be deleted?

Reply:
This was generated from [a script](https://github.com/firebase/firebase-ios-sdk/blob/master/Firebase/InAppMessaging/ProtoSupport/generate_nanopb_protos.sh). I was advised to re-generate these if there was a directory change. The file content should be the same though, since the proto hasn't changed.

Reply:
I was advised to regenerate these files upon changing the directory. The content shouldn't have changed though, since the underlying proto hasn't changed.